### DISCUSSION TITLE
Issue with project url

### DISCUSSION THREAD

**Discussion by amruttt** [github/india#806] [Students]
Hello sir I am Amrut Bhave. I am submitting my project still no reply it's showing under review because of late evaluation my project url was not open it's showing page not found and my azure also not open it's showing you can't access this site so please help me out this problem?

**Comment by ey-mentor1** [comment]
Hello,
 Can you please check the deployment, it seems the deployment for the project was dropped.
And regarding evaluation, your project will be evaluated within this week.

Can you please add the demo video demonstrating the project details and it's working and how you are using azure service in the project.

Video is required incase your deployment goes down

Thank you
